src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -10,13 +10,31 @@ namespace Categories
     using Tests.Permutations;
     using Tests.Tools;
     using Variables;
+#if NET452
     using VisualStudioDebugHelper;
+#endif
 
     public abstract class Base
     {
         public static string SessionId;
-        static readonly bool InvokeEnabled = bool.Parse(ConfigurationManager.AppSettings["InvokeEnabled"]);
-        static readonly TimeSpan MaxDuration = TimeSpan.Parse(ConfigurationManager.AppSettings["MaxDuration"]);
+        static readonly bool InvokeEnabled;
+        static readonly TimeSpan MaxDuration;
+
+        static Base()
+        {
+            try
+            {
+//                InvokeEnabled = bool.Parse(ConfigurationManager.AppSettings["InvokeEnabled"]);
```

Comment:
tbh I don't really know why this isn't working. I expected this to work as we're using the config manager package but it doesn't even work when running the tests on .net core under windows. Any ideas?

Reply:
I found this issue and it seems to be the same issue: https://github.com/dotnet/corefx/issues/22101 as the entry assembly here is testhost.dll as well.